td.vue/src/i18n/index.js

```diff
@@ -26,8 +27,8 @@ const get = () => {
     if (i18n === null) {
         i18n = new VueI18n({
             locale: 'eng',
-            messages: { ara, deu, ell, eng, id, spa, fin, fra, hin, por, zho }
-            // hide RUS & UKR for now: messages: { ara, deu, ell, eng, id, spa, fin, fra, hin, por, rus, ukr, zho }
+            messages: { ara, deu, ell, eng, id, ms, spa, fin, fra, hin, por, zho }
```

Comment:
ideally both 'id' and 'ms' would be imported in alphabetical order, so could these both be moved to after 'hi'
for both lines 30 & 31